primitives/Cargo.toml

```diff
@@ -26,6 +26,9 @@ bitvec = { version = "0.20.1", default-features = false, features = ["alloc"] }
 frame-system = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 hex-literal = "0.3.1"
 parity-util-mem = { version = "0.9.0", default-features = false, optional = true }
+thiserror = "1.0.23"
+[target.'cfg(not(target_os = "unknown"))'.dependencies]
+zstd = "0.5.0"
```

Comment:
we could make it an optional dependency and include it in `std` feature as we do for other such deps

Reply:
I sorry, I remember now there were troubles compiling zstd to wasm
https://github.com/paritytech/polkadot/pull/2288#discussion_r561949799
maybe this suggestion was wrong, but I think we can make it both optional and gated by unknown target